components/infobox/wikis/rainbowsix/infobox_league_custom.lua

```diff
@@ -0,0 +1,306 @@
+	if String.isEmpty(tier) then
+		return nil
+	end
+
+	--clean tier from unallowed values
+	--convert from text (old entries) to numbers
+	tier = Tier.number[string.lower(tier)] or tier
+	local tierText = Tier.text[tier]
+	local hasInvalidTier = tierText == nil
+	local hasTypeSetAsTier
+	--the following converts a tier that should be a tiertype
+	if hasInvalidTier then
+		local tempTypeForCheck = Tier.numberToType[tier] or tier
+		tempTypeForCheck = Tier.types[string.lower(tempTypeForCheck)]
+		if tempTypeForCheck then
+			hasTypeSetAsTier = true
+			hasInvalidTier = nil
+			if String.isEmpty(tierType) then
+				tierType = tempTypeForCheck
+			end
+		end
+	end
+	tierText = tierText or tier
+
+	local hasInvalidTierType = false
+
+	local tierDisplay = '[[' .. tierText .. ' Tournaments|' .. tierText .. ']]'
+		.. '[[Category:' .. tierText .. ' Tournaments]]'
+
+
+	if not String.isEmpty(tierType) then
+		tierType = Tier['types'][string.lower(tierType or '')] or tierType
+		hasInvalidTierType = Tier['types'][string.lower(tierType or '')] == nil
+		local tierTypeDIsplay
+		tierTypeDIsplay = '[[' .. tierType .. ' Tournaments|' .. tierType .. ']]'
+			.. '[[Category:' .. tierType .. ' Tournaments]]'
+		if not hasTypeSetAsTier or tierType ~= tierTypeDIsplay  then
+			tierDisplay = tierTypeDIsplay .. '&nbsp;(' .. tierDisplay .. ')'
+		end
+	end
+
+	tierDisplay = tierDisplay ..
+		(_args['ubisoft-sponsored'] == 'true' and _UBISOFT_ICON or '') ..
+		(hasInvalidTier and '[[Category:Pages with invalid Tier]]' or '') ..
+		(hasInvalidTierType and '[[Category:Pages with invalid Tiertype]]' or '') ..
+		(hasTypeSetAsTier and '[[Category:Pages with Tiertype set as Tier]]' or '')
+
+	Variables.varDefine('tournament_tier', tier)
+	Variables.varDefine('tournament_tiertype', tierType)
+	--overwrite wiki var `tournament_liquipediatiertype` to allow `args.tiertype` as alias entry point for tiertype
+	Variables.varDefine('tournament_liquipediatiertype', tierType)
+	return tierDisplay
+end
+
+function CustomLeague:defineCustomPageVariables()
+	--Legacy vars
+	Variables.varDefine('tournament_ticker_name', _args.tickername or '')
+	Variables.varDefine('tournament_tier', _args.liquipediatier or '')
+
+	--Legacy date vars
+	local sdate = Variables.varDefault('tournament_startdate', '')
+	local edate = Variables.varDefault('tournament_enddate', '')
+	Variables.varDefine('tournament_sdate', sdate)
+	Variables.varDefine('tournament_edate', edate)
+	Variables.varDefine('tournament_date', edate)
+	Variables.varDefine('date', edate)
+	Variables.varDefine('sdate', sdate)
+	Variables.varDefine('edate', edate)
+
+	--Apexs specific vars
+	Variables.varDefine('tournament_gamemode', string.lower(_args.mode or ''))
+	Variables.varDefine('tournament_series2', _args.series2 or '')
+	Variables.varDefine('tournament_publisher', _args['ubisoft-sponsored'] or '')
+	Variables.varDefine('tournament_ubisoft_tier', _args.ubisofttier or '')
+end
```

Comment:
tournament_tier_type
tournament_ticker_name
tournament_prizepool
tournament_mode
tournament_currency

should be defined here